native/python/py_method.cpp

```diff
@@ -336,8 +335,8 @@ void PyJPBoundMethod::__dealloc__(PyObject* o)
 	TRACE_IN("PyJPBoundMethod::__dealloc__");
 	PyJPBoundMethod* self = (PyJPBoundMethod*)o;
 
-	Py_DECREF(self->m_Instance);
-	Py_DECREF(self->m_Method);
+	Py_XDECREF(self->m_Instance);

```

Comment:
it is a little defensive, but after correcting the return value in PyJPBoundMethod::__init__ i thought that if we'd throw before setting self->m_Instance then this would crash.